packages/mendel-pipeline/src/multi-process/base-master.js

```diff
@@ -1,7 +1,8 @@
 const analyticsCollector = require('../helpers/analytics/analytics-collector');
 const analyzeIpc = require('../helpers/analytics/analytics')('ipc');
 const {fork} = require('child_process');
-const numCPUs = require('os').cpus().length;
+// After 4, there is a diminishing marginal utility at cost of memory.
+const RECOMMENDED_CPUS = Math.min(2, Math.max(4, require('os').cpus().length));
```

Comment:
So, this will be always less than or equal to 2 ?? Won't that make daemon much slower??